content/topics/starting-team/roles/index.yml

```diff
@@ -2,6 +2,7 @@ pagetitle: Roles
 description: To build and maintain a user-centred service, you need team members with a range of skills and who are open to collaborating.
 created_by: Digital Transformation Agency
 published_date: '2017-08-07'
+keywords: MDT, Delivery manager, service manager, product manager, user researcher, designer, developer,content designer, service designer, multidisciplinary team, interaction designer
```

Comment:
@sirleech re. `developer,content designer,` are the keywords being parsed as comma separated, or are you only taking spaces into account?